1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,100 @@
 // Write your code here
+const images = [
+  {
+    src: "./assets/cute-cat-a.png",
+  },
+  {
+    src: "./assets/cute-cat-b.jpg",
+  },
+  {
+    src: "./assets/cute-cat-c.jpg",
+  },
+];
+
+//variables
+const backwardBtn = document.querySelector("#backward-btn");
+const forwardBtn = document.querySelector("#forward-btn");
+const image = document.querySelector("#carousel-img");
+const autoBackBtn = document.querySelector("#auto-backward");
+const autoForwardBtn = document.querySelector("#auto-forward");
+const stopBtn = document.querySelector("#stop");
+const showTime = document.querySelector(".show-time");
+
+let ids = 0;
```

Comment:
The naming here is not too clear is there something else we could name for better clarity?